### PR TITLE
pr

### PR DESCRIPTION
**Pull request by AndriiTomalia**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FLASK_APP=server.py
+
+FLASK_ENV=development
+
+POSTGRES_USER=user
+POSTGRES_PW=password
+POSTGRES_HOST=127.0.0.1
+POSTGRES_PORT=5432
+POSTGRES_DB=db_name
```
Comment (olehkorkh-planeks) [inline .env]: Better to rename this file to something like `.env.example` and add an explanation to the `README.md` that developer needs to create `.env` manually. The file `.env` shouldn't be in the repository.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+This repository is a boilerplate Flask project for quickly getting started.
+
+## Getting started
+
+Steps:
+
+1. Clone/pull/download this repository
+2. Create a virtualenv with `python3 -m venv env_py37` and install dependencies with `pip install -r requirements.txt`
+4. Configure your `.env` variables
```
Comment (olehkorkh-planeks) [inline README.md:12]: Add here note that developer should create `.env` file first. Check my previous comment for `.env`.

File: manage.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from flask_script import Manager
+from flask_migrate import Migrate, MigrateCommand
+
+
+from src.model import db
+from src.server import server
+
+migrate = Migrate(server, db)
```
Comment (olehkorkh-planeks) [inline manage.py:8]: The variable `migrate` is not using in the code.

File: src/config.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+DEBUG = True
+TESTING = False
```
Comment (olehkorkh-planeks) [inline src/config.py]: I think that we need to add optional support for changing these variables in `.env`.

File: src/model/__init__.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+from .user import User
```
Comment (olehkorkh-planeks) [inline src/model/__init__.py]: As I understand, all new models need to be added here. In such a case, please add the comment with explanation here.

File: src/route/common.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+
+@common_blueprint.route('/routes')
+def list_routes():
+    output = []
+    for rule in current_app.url_map.iter_rules():
+        methods = ','.join(rule.methods)
+        line = "{:50s} {:20s}".format(str(rule), methods)
+        output.append(line)
+    return jsonify(routes=output)
```
Comment (olehkorkh-planeks) [inline src/route/common.py]: What is the reason for this function?

**Review by olehkorkh-planeks — CHANGES_REQUESTED** [review]
Check the comments

File: src/route/common.py (hunk 6 above)
Comment (AndriiTomalia) [reply src/route/common.py]: This function lists all available project URLs and returns them in JSON format.

Comment (olehkorkh-planeks) [reply src/route/common.py]: It is obvious, I mean what is the reason for that? In which case it could be usable?

File: manage.py (hunk 3 above)
Comment (AndriiTomalia) [reply manage.py:8]: It is required to use migration commands from manage.py  https://flask-migrate.readthedocs.io/en/latest/#installation

Comment (olehkorkh-planeks) [reply manage.py:8]: OK

**Pull request merged**